components/Error/Error.js

```diff
@@ -1,12 +1,13 @@
 import React from "react";
-import { Text, View } from "react-native";
+import { Text } from "react-native";
+import { SafeAreaView } from "react-native-safe-area-context";
 
 // Can take optional props to customize error
 export default function Error({ message }) {
   const errorText = `Error getting resource: ${message}`;
   return (
-    <View>
+    <SafeAreaView>
```

Comment:
In a previous comment I suggested using "react-native-safe-area-context" instead of the built-in solution, but I failed to describe how to use it. I'd recommend using the [useSafeAreaInsets](https://github.com/th3rdwave/react-native-safe-area-context#usesafeareainsets) hook instead of the provided component since it gives you better control over the insets

Reply:
Will update this in the next PR